context/app/static/js/components/Detail/CollectionCreatorsTable/CollectionCreatorsTable.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+
+import Paper from '@material-ui/core/Paper';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Tooltip from '@material-ui/core/Tooltip';
+
+import { StyledTableContainer, HeaderCell } from 'shared-styles/Table';
+import { HeaderIconCell, StyledLink, StyledInfoIcon } from './style';
+import SectionHeader from '../SectionHeader';
+import SectionContainer from '../SectionContainer';
+
+const useTooltipStyles = makeStyles((theme) => ({
+  tooltip: {
+    backgroundColor: theme.palette.secondary.main,
+    borderRadius: '0.4rem',
+  },
+}));
+
+function CollectionCreatorsTable(props) {
+  const { creators: tableRows } = props;
+
+  const columns = [
+    { id: 'name', label: 'Name' },
+    { id: 'affiliation', label: 'Affiliation' },
+  ];
+
+  const classes = useTooltipStyles();
+
+  return (
+    <SectionContainer id="datasets-table">
+      <SectionHeader variant="h3" component="h2">
+        Creators
+      </SectionHeader>
+      <Paper>
+        <StyledTableContainer $maxHeight={312}>
```

Comment:
Taken from the design.

Reply:
Perhaps the other way-- If there ever is any reason for a precise pixel value, make sure that's communicated. Otherwise I'll assume it's just a value that looked right. Thanks!